### DISCUSSION TITLE
When building ghdl yosys plugin, it failed in the last step of building yosys.

### DISCUSSION THREAD

**Discussion by 48004800** [YosysHQ/yosys#3784] [Q&A] [answered]
I use the Windows system. It seems that using cygwin will fail. So how do I build yosys？

**Comment by 48004800** [comment]
https://github.com/YosysHQ/yosys/pull/3448
According to the instructions on this webpage, I can build yosys separately, but I cannot build ghdl yosys plugin.

**Reply by mmicko** [reply, marked as answer]
You would need to contact maintainers of ghdl yosys plugin for that, it is project unrelated to us.
